css/qmwp-search-correlations.css

```diff
@@ -20,6 +20,9 @@
     vertical-align: bottom !important;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [vertical-align can't be used with display: block.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1163435820/issues/source?bid=2836188&fileBranchId=2916246#l20)